src/app/frontend/deploy/deployfromsettings_controller.js

```diff
@@ -0,0 +1,124 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {stateName as replicasetliststate} from 'replicasetlist/replicasetlist_state';
+
+/**
+ * Controller for the deploy from settings directive.
+ *
+ * @final
+ */
+export default class DeployFromSettingsController {
+  /**
+   * @param {!angular.$log} $log
+   * @param {!ui.router.$state} $state
+   * @param {!angular.$resource} $resource
+   * @param {!angular.$q} $q
+   * @ngInject
+   */
+  constructor($log, $state, $resource, $q) {
+    /**
+     * @export {!DeployFromSettingsController}
+     * It initializes the scope output parameter*/
```

Comment:
Add new line. And move `@export` to the end of the comment. This is to have all comments look the same.